snuba/utils/streams/backends/kafka.py

```diff
@@ -629,6 +629,19 @@ def closed(self) -> bool:
     "sasl.username",
     "sasl.password",
     "security.protocol",
+    "ssl.cipher.suites",
+    "ssl.enabled.protocols",
+    "ssl.endpoint.identification.algorithm",
+    "ssl.key.location",
+    "ssl.key.password",
+    "ssl.key.pem",
+    "ssl.keystore.location",
+    "ssl.keystore.password",
+    "ssl.keystore.type",
```

Comment:
Cannot find this one in the librdkafka settings doc.

Reply:
Thanks for the feedback. I removed these (looks like they're broker configurations, not client).